src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  color: #000;
```

Comment:
it's default value

Reply:
color isn't default value in this situation cause using for links